### PR TITLE
Fix/tendermint0.35

### PR DESCRIPTION
**Pull request by p4u**
This PR adds some fixes and improvements done during the benchmarking session of the tendermint-0.35 based vochain

### CODE REVIEW

File: vochain/start.go [hunk 1]
```diff
@@ -309,6 +304,23 @@ func newTendermint(app *BaseApplication,
 		}
 	}
 
+	// We need to fetch chain_id in order to make Replay work,
+	// since signatures depend on it.
+	log.Warnf("genesis file at %s", tconfig.GenesisFile())
```
Comment (altergui) [inline vochain/start.go]: shouldn't this be Info or Debug? (i imagine this being a leftover from debugging)

File: vochain/vochaininfo/vochaininfo.go [hunk 2]
```diff
@@ -128,21 +128,21 @@ func (vi *VochainInfo) VoteCacheSize() int {
...
 	if vi.vnode.Node != nil {
 		return vi.vnode.Node.NetInfo(context.Background())
 	}
 	return nil, fmt.Errorf("vochain node not ready")
 }
 
 // Peers returns the whole list of peers, including ID and URL
-func (vi *VochainInfo) Peers() []coretypes.Peer {
-	ni, err := vi.netInfo()
+func (vi *VochainInfo) Peers() int {
```
Comment (altergui) [inline vochain/vochaininfo/vochaininfo.go]: maybe call it NPeers then? to make it consistent with what's return'ed

**Review by altergui — COMMENTED** [review]
i didn't review the oracle changes since they are out of my current scope of understanding but i added a couple comments

File: vochain/start.go (hunk 1 above)
Comment (p4u) [reply vochain/start.go]: right

File: vochain/vochaininfo/vochaininfo.go (hunk 2 above)
Comment (p4u) [reply vochain/vochaininfo/vochaininfo.go]: yep

**Pull request merged**